### DISCUSSION TITLE
Simplest command(s?) to format code AND organise imports like Prettier?

### DISCUSSION THREAD

**Discussion by cyrus01337** [biomejs/biome#3523] [Q&A] [answered]
Currently I use the following I cobbled together after scouring the documentation, however, it feels (and probably is) way too long.
```sh
biome format --write --skip-errors --no-errors-on-unmatched --changed --since=main --log-level=none
biome lint --write --unsafe --skip-errors --no-errors-on-unmatched --changed --since=main --log-level=none
```
The end goal is to get as close to how Prettier + import sort plugin behaves as reasonable. Any thoughts?

**Comment by Conaclos** [comment, marked as answer]
`biome check` executes all Biome tools (linter, formatter, and the import sorter). You can disable the linter using `--linter-enabled=false`.

If you use `write`, I am not sure that `--skip-errors` is so useful…
Also, do you really need to specify `--since`?

```shell
biome check --linter-enabled=false --write --no-errors-on-unmatched --changed --log-level=none
```

**Reply by ematipico** [reply]
You can also remove `--log-level`, because it's `none` by default

**Reply by cyrus01337** [reply]
`--since` I included for empty efficiency and can be omitted. I'll give `biome check ...` a try.

**Reply by cyrus01337** [reply]
Works great, thanks so much both! :slightly_smiling_face:
